src/GoTrueClient.ts

```diff
@@ -2085,11 +2081,11 @@ export default class GoTrueClient {
       // finished and tests run endlessly. This can be prevented by calling
       // `unref()` on the returned object.
       ticker.unref()
-      // @ts-ignore
+      // @ts-expect-error TS has no context of Deno
```

Comment:
Is this needed here, since the Deno scenario is covered below? Or maybe the Deno check should happen first?

Reply:
IMO `@ts-expect-error` is a much better way to document why a typescript error needs to be ignored compared to `@ts-ignore`, I simply replaced the existing override with this one.

Reply:
I don't know of a way to check for Deno without TS complaining